target/linux/ramips/dts/mt7620a_sitecom_wlr-4100v1002.dts

```diff
@@ -0,0 +1,184 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "sitecom,wlr-4100v1002", "ralink,mt7620a-soc";
+	model = "Sitecom sw66 WLR-4100 v1 002";
```

Comment:
What's with the stray sw66?

Reply:
Sorry, It is a refusal of several "*.dts" file versions